roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -443,10 +448,16 @@ description of the flow. See module-docs for full details.
   - **Backable** is when the necessary backing statements have been acquired but
     those statements and the parablock candidate haven't been backed in a relay
     chain block yet.
-- **Fragment tree:** A parachain fragment not referenced by the relay-chain.
-  It is a tree of prospective parachain blocks.
+- **Fragment:** A prospective para block (that is, a block not yet referenced by
+  the relay-chain). Fragments are anchored to the relay-chain at a particular
+  relay-parent.
+- **Fragment tree:**
+  - A tree of fragments. Together, these fragments define one or more
+    prospective paths a parachain's state may transition through.
+  - See the "Fragment Tree" section.
```

Comment:
Should be:

```suggestion
  - See the "Fragment Tree" section on the Prospective Parachains page.
```

ideally with a hyperlink to that page's section. But, [this PR](https://github.com/paritytech/polkadot/pull/6933) hasn't been merged yet, so this change should probably go in that PR.

Reply:
Or, we can have a shared glossary for the backing subsystems, i.e. a separate Glossary subpage. I personally think it's nice to have a glossary on the same page so readers don't have to hunt for it, and to have terms only relevant to what they're reading about, but it's an idea.

Reply:
Makes sense that it should go in the prospective parachains PR 👍

Reply:
Thanks for the review!